src/main/java/demurebot/DemureBot.java

```diff
@@ -0,0 +1,96 @@
+package demurebot;
+
+import demurebot.command.Command;
+import demurebot.storage.Storage;
+import demurebot.task.TaskList;
+import demurebot.ui.Parser;
+import demurebot.ui.Ui;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Main class for the DemureBot application.
+ * This class handles the initialization and execution of the chatbot.
+ */
+public class DemureBot {
+    private TaskList list;
```

Comment:
Perhaps using a noun in the variable would be clearer to users. makes it less ambiguous to readers